packages/fxa-content-server/app/tests/spec/views/app.js

```diff
@@ -324,16 +365,43 @@ describe('views/app', function () {
       });
     });
 
-    describe('with a view that errors', function () {
-      var renderError = AuthErrors.toError('UNEXPECTED_ERROR');
+    describe('with a survey', function () {
+      var ViewThatRenders = Backbone.View.extend({
+        afterVisible: sinon.spy(),
+        destroy: sinon.spy(),
+        logView: sinon.spy(),
```

Comment:
I cannot find the assertions where these spies are used.

Reply:
These each just need the `afterVisible` and `logView` functions passed in order for the view to render. They don't need to be spy they could be noop functions, I just copied from another test.